CMakeLists.txt

```
@@ -40,6 +40,10 @@ option(LTO "Enable Link Time optimization (Requires Release build, only works wi
 
 string(TOUPPER "${CMAKE_BUILD_TYPE}" CONFIG)
 
+if(WIN32 AND (PROFILE OR VALGRIND_PROFILE OR GCOV OR STATIC_LINK_VW OR BUILD_JAVA OR LTO))
+  message(FATAL_ERROR "Unsupported option enabled on Win32 build")
```

[Minor] Are we actually building in Win32 mode? (X86) If not, consider changing to "on Windows build"

```suggestion
  message(FATAL_ERROR "Unsupported option enabled on Windows build")
```